nats-server/connz.yml

```diff
@@ -0,0 +1,20 @@
+type: json
+data:
+  url: /connz
+  server: ${NATS_SERVER_HOSTPORT}
+
+observation:
+  - name: connz
+    metricNamespace: nats_server
+    path: $.
+
+    metric:
+      - name: connections
+        source: num_connections
+        type: long_gauge
+        label: connections
+        description: The number of connections
+
+    tag:
+      - name: nats.servers.server
```

Comment:
tag name can be `nats.server.id`. `nats.servers.server` seems to stutter

Reply:
OK, DONE.

Reply:
Hi Tien, Bojan and I had a discussion. Shall we change the metric namespace to `nats` and prefix all metric names with `server`? For the streaming server (when we add the integration) we can prefix the metric namespace with `streaming.server`

The metric name will be like `server. connections`, `server.subscriptions.removes` etc